Makefile

```diff
@@ -54,3 +55,17 @@ ut-circle: calico_mesos_builder.created dist/calico_isolator
 	--with-xunit --xunit-file=/circle_output/output.xml; RC=$$?;\
 	[[ ! -z "$$COVERALLS_REPO_TOKEN" ]] && coveralls || true; exit $$RC'
 
+builder-rpm:
+	cd $(WD)/packages && docker build -t calico-mesos-builder .
+	mkdir -p $(WD)/packages/rpms
+	mkdir -p $(WD)/packages/srpms
+	cd $(WD)/packages && docker run --name=calico-mesos-builder1 -v $(WD)/packages/rpms:/opt/rpms -t calico-mesos-builder cp -r /root/rpmbuild/RPMS /opt/rpms
+	cd $(WD)/packages && docker run --name=calico-mesos-builder2 -v $(WD)/packages/srpms:/opt/srpms -t calico-mesos-builder cp -r /root/rpmbuild/SRPMS /opt/srpms
```

Comment:
I think that `-t` flag is incorrect, for `docker run` that sets tty to true or false. So try with that removed. Also, I think we can create rpms and srpms in one go:
```
cd packages && docker run --name=calico-mesos-builder \
-v packages/rpms:/opt/rpms \
-v packages/srpms:/opt/srpms \
bash -c 'cp -r /root/rpmbuild/SRPMS /opt/srpms && cp -r /root/rpmbuild/RPMS /opt/rpms'
```

Or even better, if there isn't too much extra stuff in `/root/rpmbuild`:
cp -r /root/rpmbuild /opt/